Jenkinsfile

```diff
@@ -0,0 +1,78 @@
+pipeline
+{
+    agent
+    {
+        node
+        {
+            label 'embedded-docker'
+        }
+    }
+    environment
+    {
+        MAKEFLAGS = '-j 3'
+        PACKAGE_NAME = 'um_kernel'
+        DOCKER_CONTAINER_NAME = 'um-kernel_build'
+        DOCKER_IMAGE_NAME = 'um-kernel_image'
+        BUILD_OUTPUT_DIR = '_build_armhf'
+    }
+    stages
+    {
+        stage('Prepare')
+        {
+            steps
+            {
+                echo "Starting with a clean environment"
+                step([$class: 'WsCleanup'])
```

Comment:
this should be documented as a hack/workaround as we really do not want to clean and checkout as part of our normal process